README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
The inability to spend gnuts elsewhere does not change the fact that you are losing value by the interest rate. For example, if I buy gold, my money becomes less liquid. By holding gold I am forgoing the possibility to take advantage of a sudden change in prices, like a sale on pinto beans.
Similarly, if I am holding gnuts, I am forgoing the ability to take advantage of a sale. When value is inside of a less-liquid location, the owner is losing value by the interest rate. If you don't pay to cover his cost, he wont invest.

If we can't spend from the reserve pool, then the money is already deleted. There is no cost to draining it for a retirement attacker.
The cost of refilling an empty pool is carried by the honest validators who remain.

If control over the reserve pool is proportionate to how many Gnuts you own, then we need to examine the combined ownership of both cryptoeconomically.
If this combo-asset is doing both things as once, then it must be doing them both poorly.
Too many bonds for stopping double-singing, but too few to stop retirement attacks.

just because you balance marginal cost and marginal return doesn't mean it is a cryptoeconomically sound design. Reusing the same asset for these 2 different goals is a bad idea. Each goal puts different restrictions on the bonds. The combination of restrictions is much worse than either on it's own.

If an amount of value equal to 1/2 (the value of the reserve pool + the total value of the gnuts) should ever flow through a single block, it will be profitable for a coalition to double-sign. This is a retirement attack. They don't care about getting slashed, because they are stealing more than they can lose.

If you make the reserve pool bigger, you will increase security against retirement attacks. If you make the pool smaller, the blockchain will be more affordable to run.

Using the same bonds for both these things makes it so you can't optimize both, you are finding a semi-bad medium between 2 bad alternatives.